be/src/agent/publish_version.cpp

```diff
@@ -70,14 +70,8 @@ void run_publish_version_task(ThreadPoolToken* token, const TPublishVersionReque
     if (is_replication_txn) {
         std::vector<std::vector<TTabletId>> partitions(num_partition);
         for (size_t i = 0; i < publish_version_req.partition_version_infos.size(); i++) {
-            Status st = StorageEngine::instance()->replication_txn_manager()->get_txn_related_tablets(
+            StorageEngine::instance()->replication_txn_manager()->get_txn_related_tablets(
```

Comment:
Why remove the log ?

Reply:
now get_txn_related_tablets just get some data from memory, will not fail, so remove the log

Reply:
But the function have return status